lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -602,6 +612,10 @@ def from_ansible_params(ap, old_service):
         s.update_order = ap['update_order']
         s.user = ap['user']
 
+        s.command = ap['command']
+        if isinstance(s.command, string_types):
+            s.command = shlex.split(s.command)
```

Comment:
You might want to add an `elif` and check that the command is indeed a list of strings (and if not, fail).

Reply:
👍